src/main/java/net/coderbot/iris/mixin/MixinWorldRenderer.java

```diff
@@ -45,18 +50,28 @@
 	private static final String RENDER_WEATHER = "Lnet/minecraft/client/render/WorldRenderer;renderWeather(Lnet/minecraft/client/render/LightmapTextureManager;FDDD)V";
 	private static final String RENDER_WORLD_BORDER = "Lnet/minecraft/client/render/WorldRenderer;renderWorldBorder(Lnet/minecraft/client/render/Camera;)V";
 
+	@Unique
+	HorizonRenderer horizonRenderer = new HorizonRenderer();
```

Comment:
This can probably be private?

Reply:
Sure, didn't think of much reason to keep it private but again, not too familiar with accessing.

Reply:
Fixed.

Reply:
Yeah, it's best to keep fields and similar things private unless they need to be public, that's part of encapsulation.